### ISSUE TITLE
Add Zotero CSL exporter for markdown citation markup

### ISSUE THREAD

**Issue by danmackinlay** [chrisjsewell/ipypublish#19]
Citations like this
```html
<cite data-cite="kirkeminde_thermodynamic_2012">(Kirkeminde, 2012)</cite> 
```
are sometimes desirable for online notebook presentations. Zotero can be made to produce them using [CSL](http://citationstyles.org/), if not with stable BibTex keys (see #2 )

**Comment by danmackinlay** [comment]
I'm clocking off for today, but intend to submit a PR with the appropriate CSL when I've met some other deadlines.

**Comment by chrisjsewell** [comment]
@danmackinlay check out the new update: https://github.com/chrisjsewell/ipypublish/blob/master/ipypublish/preprocessors/latextags_to_html.py
and the slides it outputs from the example: https://chrisjsewell.github.io/ipypublish/Example.slides.html#/2
and then tell me how amazing I am haha.

**Comment by danmackinlay** [comment]
This is simultaneously 

1. an incredible feature that I have longed for for ages, and that we should tell the ``nbconvert`` folks about
2. an example of my darkest fears about how I get distracted from writing my thesis, but improving my thesis writing workflow. 

Regardless, I think you just negated the need for my CSL hack...

**Comment by chrisjsewell** [comment]
hahahaha yeh I know, I really need to get back to my research lol.
I just want to add hyperlink anchors and captions to the output tables/figures (from the latex_doc meta_data tags) and that's basically me done with what my original vision was for this project

**Comment by danmackinlay** [comment]
In light of #2 I'm going to experimentally re-open this one.
I've written CSL styles using the [csl style visual editor](http://editor.citationstyles.org/visualEditor/) that will output the supported formats.

```html
<cite data-cite="Author1997">Author, 1997<cite>
```
and
```latex
\cite{Author1997}
```

Here is the [bibtex cite](https://github.com/danmackinlay/styles/blob/master/bibtex-cite-command.csl) one and [here is the html one](https://github.com/danmackinlay/styles/blob/master/html.csl)

You can install these into Zotero and use them as standard citation export formats, just like "APA" or "Cell" or whatever.

However, they don't have access to bibtex cite key, so some hackery (or guessing) means this isn't a 100% solution yet.

**Comment by chrisjsewell** [comment]
Sounds good, I'm currently hanging on to Zotero 4 for however long I can lol

**Comment by chrisjsewell** [comment]
closed in #67 

`ipubpandoc` allows for \cite{label} :cite:`label` @label and <cite data-cite="label"></cite> to all be used in the same document. See https://ipypublish.readthedocs.io/en/latest/markdown_cells.html
